aptos-move/framework/aptos-framework/sources/stake.spec.move

```diff
@@ -61,6 +62,38 @@ spec aptos_framework::stake {
         ensures exists<ValidatorFees>(aptos_addr);
     }
 
+    spec initialize_validator(
+        account: &signer,
+        consensus_pubkey: vector<u8>,
+        proof_of_possession: vector<u8>,
+        network_addresses: vector<u8>,
+        fullnode_addresses: vector<u8>,
+    ){
+        let pubkey_from_pop = bls12381::spec_public_key_from_bytes_with_pop(
+            consensus_pubkey,
+            proof_of_possession_from_bytes(proof_of_possession)
+        );
+        aborts_if !option::spec_is_some(pubkey_from_pop);
+        let addr = signer::address_of(account);
+        let post_addr = signer::address_of(account);
+        let allowed = global<AllowedValidators>(@aptos_framework);
+        aborts_if exists<ValidatorConfig>(addr);
+        aborts_if exists<AllowedValidators>(@aptos_framework) && !vector::spec_contains(allowed.accounts, addr);
+        aborts_if stake_pool_exists(addr);
+        aborts_if exists<OwnerCapability>(addr);
```

Comment:
Where does `aborts_if exists<OwnerCapability>(addr);` come from?

Reply:
it comes from stake.move #L508 -> 545